frb_spectral_search/acf_func.py

```diff
@@ -180,7 +181,7 @@ def plot_bands(all_acf, bands, remove_nan=True):
     plt.xlabel('Freq. [MHz]', fontsize =16)
     plt.ylabel("f_dc [MHz]", fontsize =16)
     plt.legend()
-def wrapper(noise_spec, spectra, n, noise_std, skip): 
```

Comment:
Meaning of this is pretty vague....please document it if it's important, or remove if it's exploratory.